package.json

```diff
@@ -64,57 +25,66 @@
     "test": "concurrently -k -s first \"yarn start:test\" \"wait-on http://localhost:3112 && yarn cypress:run\"",
     "cypress:run": "env-cmd -f ./.env.test cypress run --browser chrome"
   },
-  "browserslist": {
-    "production": [
-      ">0.2%",
-      "not dead",
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
+  "dependencies": {
+    "@emotion/react": "11.10.6",
+    "@emotion/styled": "11.10.6",
+    "@graasp/chatbox": "github:graasp/graasp-chatbox#main",
+    "@graasp/query-client": "github:graasp/graasp-query-client#main",
+    "@graasp/sdk": "github:graasp/graasp-sdk#92-add-build-utilities-for-extra",
```

Comment:
To update ;)

Reply:
Need a review on https://github.com/graasp/graasp-sdk/pull/93